sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+## CNCF Runtime SIG Charter
+
+
+### Primary Author: Quinton Hoole
+
+2019-09-02
+
+Also reviewed and contributed to by:
+
+*   Liz Rice
+*   Brian Grant
+
+## Introduction
+
+This is the charter referred to in “[CNCF
+SIGs](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#sig-charter)”
+by the CNCF TOC, and consistent with the [proposed SIG
+definition](https://github.com/cncf/toc/blob/master/sigs/proposed.md).
+
+
+## Areas Considered In Scope
+
+Workload execution and management systems, components and interfaces
+used in [modern cloud-native
+environments](https://github.com/cncf/toc/blob/master/DEFINITION.md)
+are in scope, including:
+
+
+
+*   generalized orchestration, autoscaling, scheduling, execution, container
+    runtimes, sandboxing, virtualization, image packaging and distribution and specialized architectures thereof, e.g. 
+    *   those aimed specifically at Edge, IoT, Batch, Big Data, AI/ML, etc
+    *   those incorporating specialized computing elements beyond CPUs, including GPUs, TPUs, FPGAs, ASICs, etc.
+
+We strive to understand the fundamental characteristics of different approaches with respect to availability,
+scalability, performance, consistency, ease-of-use, cost and operational complexity; and relate these to their
+suitability to various cloud-native use cases.
+
+
+## Areas Considered Out Of Scope
+
+Anything not considered in scope above is out of scope.  See also “Interfaces with Related Groups” below.
+
+Examples include:
+
+
+
+*   General authentication, authorization, accounting, auditing, etc (even though these clearly apply to )
+    - because AAA, etc is clearly the domain of the [CNCF Security SIG](https://github.com/cncf/sig-app-delivery).
```

Comment:
this should read SIG App Deliver, not Security SIG

Reply:
Yup.

Reply:
Fixed.